Example_Programs/programming_concpets/ttc_1a/src/c_div.c

```diff
@@ -0,0 +1,19 @@
+#include "header_def.h"
+
+int div(complex_num s1, complex_num s2){
+    struct complex_num res;
+
+    if(s2.real == 0 || s2.img == 0 ){
+        return -1;
+        printf("You cannot divide by 0");
+    }
+
+    int a=(((s1.real)*(s2.real))+((s1.img)*(s2.img)))/(pow(s2.real,2)+pow(s2.img,2));
+    int b=(((s2.real)*(s1.img))-((s1.real)*(s2.img)))/(pow(s2.real,2)+pow(s2.img,2));
+
+    res.real = a;
+    res.img = b;
+    printf("The division is %d + i%d\n", a,b);
```

Comment:
Instead add debug print using some macros. User can enable or disable debug prints.

Reply:
This code was not for the purpose of debugging, this was supposed to be output to the end-user.(deprecated code source)